src/tests/class_tests/openms/source/SvmTheoreticalSpectrumGenerator_test.cpp

```diff
@@ -95,7 +95,13 @@ START_SECTION(void simulate(PeakSpectrum &spectrum, const AASequence &peptide, b
   PeakMap exp;
   MzMLFile mz_file;
 
+#if OPENMS_BOOST_VERSION_MINOR < 56
   mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test.mzML"),exp);
+  TEST_EQUAL(spec.size(), 7);
+#else
+  mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test_boost58.mzML"),exp);
+  TEST_EQUAL(spec.size(), 8);
+#endif
```

Comment:
ok so the difference is only related to boost and not because a different LibSVM is used? I am asking because I think I remember a failing SVMWrapper test in the past. For a different version of libSVM, a prediction value of 0 got assigned to either class -1 or class 1 depending on the version.